src/currencyservice/server.js

```diff
@@ -14,11 +14,22 @@
  * limitations under the License.
  */
 
+const pino = require('pino');
+const logger = pino({
+  name: 'currencyservice-server',
+  messageKey: 'message',
+  formatters: {
+    level (logLevelString, logLevelNum) {
+      return { severity: logLevelString }
+    }
+  }
+});
+
```

Comment:
**Suggestion:** (feel free to ignore)
Feel free to create a separate `logger.js` file (in each Node.js microservice) where we initialize the `logger = pino(...)` object (just once for each microservice) and re-use that object in other files (by importing the object from the `logger.js` file) but that's a bonus (out-of-scope) for this pull-request.

Reply:
Yeah, in the beginning I was going to do it this way, but creating multiple logger files for different services seemed a bit too much so I only made changes in the required services. 
If you have a suggestion where we only use one `logger.js` and that can be used across all the services then do tell me about it. Because the only change in the logger is the name of the service, if we know how to set the name of the logger's service dynamically then it should be good. 
If that's not possible then maybe you can raise another issue for this and I'll create separate `logger.js` files for all the services.

Reply:
For now, I've created `logger.js` for paymentservice only.
Please check and validate that

Reply:
Thanks for going the extra mile (but know that you don't _have_ do)! The `logger.js` file of the paymentservice should live inside `/src/paymentservice`.